sklearnex/dispatcher.py

```diff
@@ -52,11 +52,16 @@ def get_patch_names():
     return list(get_patch_map().keys())
 
 
-def patch_sklearn(name=None, verbose=True):
+def patch_sklearn(name=None, verbose=True, _global=False):

```

Comment:
bad, strong interface change. move common code, for example:
```python
def patch_sklearn(name=None, verbose=True):
   detail_patch_sklearn(name=name, verbose=verbose, global=False)

def detail_patch_sklearn(name, verbose, global):
   ....
```

Reply:
agree, thanks)